README.md

```diff
@@ -53,7 +53,7 @@ After enabling the plugin, you need to use the `Dolby.io Subsystem` node in Blue
 
 The following steps present how to create a sample application that uses the plugin:
 
-1. Open your Unreal Editor and locate the `Blueprints` folder in the `Content Browser` window. Double-click the folder and select the `FirstPersonCharacter` Blueprint. This step should open the Event Graph for the selected Blueprint.
+1. Open your Unreal Editor and locate the `Blueprints` folder in the `Content Browser` window. Open the folder and select the `FirstPersonCharacter` Blueprint. This step should open the Event Graph for the selected Blueprint.
```

Comment:
Technically just selecting the Blueprint will not open the Event Graph :) "open the ... Blueprint" is more accurate.